### ISSUE TITLE
add VGG weights for ecoset

### ISSUE THREAD

**Issue by hahahannes** [ViCCo-Group/THINGSvision#36]
add a custom model for VGG without batch norm

**Comment by DiGyt** [comment]
Hello @hahahannes. Is there any progress on this Issue?

I'm part of Tim Kietzmann's lab and am currently working on making the Ecoset models accessible to THINGSvision. I see that converting the TF1 models provided in ecoset to TF2 in order to work with THINGSvision is quite a challenge. Therefore I wanted to reach out to you and see whether there exists any previous work and/or knowledge that can help us to get the Ecoset models into THINGSvision quickly.

**Comment by hahahannes** [comment]
Hi Dirk :)

I am currently training a VGG16 version (there is already a VGG16 batch normalization model trained on ecoset included in things vision).

I had a quick look into the pretrained models from ecoset but was also not sure how to include them as by now we only support Keras models for extraction with tensforflow.

**Comment by hahahannes** [comment]
The easiest way to include a model into things vision would be to include it into the /thingsvision/custom_models directory and to define the model and weights download there :)

**Comment by DiGyt** [comment]
Hey Hannes,

Thanks for the quick answer.

> The easiest way to include a model into things vision would be to include it into the /thingsvision/custom_models directory and to define the model and weights download there :)

Yes, that sounds reasonable. We will try to implement it like this. Anyhow, good to know that there are already TF2-based Ecoset models on the way... :)

**Comment by LukasMut** [comment]
Hi Dirk! Is there any progress on this? I've completely refactored THINGSvision and made everything more modular. Contributing should now be a little bit easier. You can shoot me a message anytime.

**Comment by LukasMut** [comment]
I reckon it's possible to port TF2 weights to Keras. Have a look into ONNX.
